test/instrumentation/modules/graphql.test.js

```diff
@@ -26,7 +26,11 @@ var test = require('tape')
 
 var mockClient = require('../../_mock_http_client')
 
-if (semver.lt(graphqlVer, '16.0.0')) {
+// See explanation of these in "lib/instrumentation/modules/graphql.js".
+const onlySupportsPositionalArgs = semver.lt(graphqlVer, '0.10.0')
```

Comment:
if we end up needing to do this in a third module it might be worth getting this logic into some sort of helper function.  Just a comment, fine for this PR.